metrics-core/src/main/java/software/amazon/swage/metrics/jmx/sensor/DiskUsageSensor.java

```diff
@@ -60,8 +62,8 @@ public void sense(final MetricContext metricContext) throws SenseException
 
             long total = fileStore.getTotalSpace();
             long free = fileStore.getUsableSpace();
-            double percent_free = 100.0 * ((double)(total-free)/(double)total);
-            metricContext.record(DISK_USED, percent_free, Unit.PERCENT);
+            double percentFree = 100.0 * (double) (total - free) / (double) total;
```

Comment:
Having 'redundant' parens make this much easier to grok at a glance, and helps show the intent.  Less easy to get it wrong when it's spelled out explicitly with grouping.

Which is to say, can you keep using extra parens here?

As for the camelCase vs underscore_case, I don't actually like camelCase better for private vars, but that's a battle lost to Java long ago.

Reply:
When in Rome. I prefer underscores, also.